src/features/VoltageConfiguration/VoltageConfiguration.tsx

```diff
@@ -21,24 +21,36 @@ interface VoltageConfigurationProps {
     pmicPort: number;
     voltageMin: number;
     voltageMax: number;
+    pmicPortLabel?: string;
+    pmicPortDescription?: string;
 }
 
 const VoltageConfiguration = ({
     pmicPort,
     voltageMin,
     voltageMax,
+    pmicPortLabel,
+    pmicPortDescription,
 }: VoltageConfigurationProps) => {
     const dispatch = useDispatch();
     const voltage = useSelector(getPmicConfigValue(pmicPort)) ?? voltageMin; // Default to voltageMin
 
+    const label = pmicPortLabel ?? `Voltage port ${pmicPort}`;
+    const description =
+        pmicPortDescription ??
+        `Set voltage for PMIC port ${pmicPort} (${label})`;
+
     return (
         <Card
             title={
                 <div className="tw-flex tw-content-between">
```

Comment:
What is the use of flex here?

Reply:
I probably need to fix some of the flex stuff, but I won't do that for this PR. I'll make a task to look into it.